pkg/ivd/ivd_protected_entity_type_manager.go

```diff
@@ -58,7 +58,7 @@ func NewIVDProtectedEntityTypeManager(params map[string]interface{},
 	logger.Infof("Loading new IVD Protected Entity Manager")
 	err := retVal.ReloadConfig(context.Background(), params)
 	if err != nil {
-		return nil, err
+		return &retVal, err
```

Comment:
The expectation is whoever calls `NewIVDProtectedEntityTypeManager` ignores error, basically, it implies that if there were some error during ivd petm initialization, save the ivd petm and just log the error. The comment in direct_protected_entity_manager.go in this PR explains it.

Reply:
Let's not do that.  That's a bad code smell and it will confuse people looking at the code.  Just log the error here and then return nil for the error.  If we ever run into errors that will actually make the IVDPETM unusable forever there's no good way to return it.  The other option would be to add another error return that is "recoverableErr" or something like that.

Reply:
Done.